### PR TITLE
[v0.2] String to Characters

### PR DESCRIPTION
**Pull request by matheushf**
- Implement stringCharsToArray function

### CODE REVIEW

File: textalyze.js [hunk 1]
```diff
@@ -17,7 +18,26 @@ function itemCounts(array) {
...
+result = itemCounts(array);
+console.log(result);
+
+/**
+ * Takes an arbitrary String as input
+ * and returns an Array with the counting of all the characters in the string
+ * @param {String} arbitrary string
+ * @returns {Array} a Map with the counting of each char
+ */
+function stringCharsToArray(string) {
```
Comment (jfarmer) [inline textalyze.js:38]: Put function definitions first, before any free-standing code.

File: textalyze.js [hunk 2]
```diff
@@ -17,7 +18,26 @@ function itemCounts(array) {
...
+  const arrayString = string.split('');
+  return itemCounts(arrayString);
```
Comment (jfarmer) [inline textalyze.js:40]: The function you implemented isn't what you've described in the comments.  This returns a `Map` of counts, not the array of things-to-be-counted.

There's no reason for `stringCharsToArray` to know about `itemCounts`.  The code can compose the two functions if that's called for.

So, the comment is the way to go and the behavior of the function should be changed.

**Review by jfarmer — APPROVED** [review]
Again, please write good commit messages!  "test", "foo", "did X", are not helpful to other people (or yoyr future self).

File: textalyze.js (hunk 1 above)
Comment (matheushf) [reply textalyze.js:38]: Hmm sorry I think I didn't understand

**Pull request merged**